packages/forma-36-react-components/src/components/Table/TableRow/TableRow.css

```diff
@@ -1,6 +1,10 @@
 @import 'settings/colors';
 
 tbody .TableRow:hover {
+  background-color: var(--color-element-lightest);
+}
+
+.TableRow--selected {
   background-color: var(--color-blue-lightest);
   box-shadow: inset -2px 0 0 var(--color-blue-mid);
```

Comment:
box shadow is not working on Safari for some reason.

<img width="587" alt="Screenshot 2021-01-14 at 10 29 48" src="https://user-images.githubusercontent.com/6163988/104571517-77603480-5653-11eb-855b-a4bf48fde96a.png">

Reply:
I know, Webkit doesn't support `box-shadow` on `<tr />`.  I tried with a pseudo element instead but that is against the spec and doesn't work at all in any browser. I haven't been able to think of any alternatives.

Reply:
I think it's fine to have this regression in Safari.

Reply:
Oh wow,TIL. Thanks 🙏